web/ui/src/app/lineage/graph/graph-builder.ts

```diff
@@ -34,13 +37,74 @@ export function lineageToGraph(lineage: IDataLineage,
         visibleOperations: IOperation[] = operationsByVisibility.true,
         hiddenOperations: IOperation[] = operationsByVisibility.false,
         hiddenOpIds: string[] = _.map(hiddenOperations, "mainProps.id"),
-        visibleNodes = visibleOperations.map(op => new RegularVisNode(op)),
+        visibleNodes = visibleOperations.map(op => new RegularVisNode(op, getLabel(op, expressionRenderService))),
         visibleEdges = createVisibleEdges(lineage, hiddenOpIds)
 
     return new VisModel(
         new vis.DataSet<VisNode>(visibleNodes),
         new vis.DataSet<VisEdge>(visibleEdges))
 }
+/**
+ * Get label from an operation
+ * 
+ * @param operation the operation where the label should be retrieved
+ * @param expressionRenderService service to render expressions
+ * 
+ * @returns the label from the operation
+ */
+export function getLabel(operation : (IOperation), expressionRenderService : ExpressionRenderService){
+    let label = operation.mainProps.name + "\n"; 
+    switch(typeOfOperation(operation)){
+        case "Aggregate":
+            let aggregations = (<any> operation).aggregations;
+            label += Object.keys(aggregations).join(",");break;
```

Comment:
`break` should be on a new line.
Having it in the end of the previous line makes it very easy to overlook when reading the code. If you want to keep the `switch `-block  more compact consider moving it in a separate function and use `case ... : return ...` pattern, concatenating strings outside.

Reply:
ok ok. I am used to an autoformat on save that was obviously not here.